src/renderer/scss/component/_card.scss

```diff
@@ -354,11 +350,19 @@
 
   @media only screen and (min-width: $medium-breakpoint) {
     .card {
-      width: calc((100% / 4) - 12px - 20px);
```

Comment:
-12 - 20 = -32, not -31. Was this intentional?

Reply:
Yeah it was a little off